src/helpers/network/getContractName.ts

```diff
@@ -0,0 +1,31 @@
+import { useSnapshot } from 'valtio'
+import ContractMetadataStore from 'stores/ContractMetadataStore'
+import Network from 'models/Network'
+import networks from 'networks'
+import prettifyContractName from 'helpers/network/prettifyContractName'
+
+export default function (
+  address: string,
```

Comment:
Why don't we put everything into an object? It will make easier for us to use this function